packages/core/android/src/main/java/io/sentry/react/RNSentryMapConverter.java

```diff
@@ -139,19 +205,7 @@ private static void addValueToWritableMap(WritableMap writableMap, String key, O
 
   public static ReadableMap jsonObjectToReadableMap(JSONObject jsonObject) {
     Map<String, Object> map = jsonObjectToMap(jsonObject);
-    return mapToReadableMap(map);
-  }
-
-  public static ReadableMap mapToReadableMap(Map<String, Object> map) {
-    // We are not directly using `convertToWritable` since `Arguments.createArray()`
-    // fails before bridge initialisation
-    Object[] keysAndValues = new Object[map.size() * 2];
-    int index = 0;
-    for (Map.Entry<String, Object> entry : map.entrySet()) {
-      keysAndValues[index++] = entry.getKey();
-      keysAndValues[index++] = entry.getValue();
-    }
-    return JavaOnlyMap.of(keysAndValues);
+    return (WritableMap) convertToJavaWritable(map);
```

Comment:
I see that base on the current code the return value can't be null, but rather than forcing the compiler to the non-null value, let's check the output of `convertToJavaWritable` and if null we can return new empty `JavaReadableMap`.